lib/defopt.py

```diff
@@ -196,14 +202,21 @@ def run(funcs: Union[Callable, List[Callable], Dict[str, Callable]], *,
         dashes) to letters, to use as alternative short flags.  Defaults to
         `None`, which means to generate short flags for any non-ambiguous
         option.  Set to ``{}`` to completely disable short flags.
+    :param options:
+        If `all`, all parameters are converted into command-line flags. If
```

Comment:
Use single quotes, not backquotes, to delimit strings (all, has_default, kwonly).  Two spaces after dots.